languages/rust/exercises/concept/string-vs-str/tests/csv_builder.rs

```diff
@@ -0,0 +1,74 @@
+use csv_builder::*;
+
+#[test]
+fn test_no_escaping() {
+    let mut builder = CSVBuilder::new();
+
+    builder.add("ant");
+    builder.add("bat");
+    builder.add("cat");
+
+    let list = builder.build();
+    //Note that from now on we can do nothing now with builder.
+    assert_eq!("ant,bat,cat", &list);
+}
+
+#[test]
+#[ignore]
+fn test_quote() {
+    let mut builder = CSVBuilder::new();
+
+    builder.add("ant");
+    builder.add("ba\"t");
+    builder.add("cat");
+
+    let list = builder.build();
+    //Note that from now on we can do nothing now with builder.
+    assert_eq!("ant,\"ba\"t\",cat", &list);
```

Comment:
```suggestion
    assert_eq!(r#"ant,"ba""t",cat"#, &list);
```

Using raw strings makes this simpler to read. Adding a second double-quote within "bat" makes it more correct.